src/bin/dual-iir.rs

```diff
@@ -200,7 +200,7 @@ const APP: () = {
             stabilizer.cycle_counter,
             env!("CARGO_BIN_NAME"),
             stabilizer.net.mac_address,
-            net::parse_or_default_broker(option_env!("BROKER")),
+            net::parse_ipv4(option_env!("BROKER").unwrap_or("10.34.16.10")),
```

Comment:
If all `parse_ipv4()` does is call the `parse()` function, why not just inline as:
```rust
option_env!("BROKER").unwrap_or("10.34.16.10").parse().unwrap()
```

Reply:
Ah right! I had thought I had to wrestle to get some trait into scope.